filament/include/filament/MaterialInstance.h

```diff
@@ -65,6 +65,11 @@ class UTILS_PUBLIC MaterialInstance : public FilamentAPI {
      */
     Material const* getMaterial() const noexcept;
 
+    /**
+     * @return the name associated with this instance if one exists, otherwise null
```

Comment:
I wonder if we shouldn't have it return `getMaterial().getName()` instead of `null`, this way users wouldn't have to check for null before printing, etc...

Reply:
I thought of this, but I was worried that a user might need the ability to determine whether or not the instance has an explicitly assigned name.

Reply:
I think it would be relevant only if you could set the name? I mean otherwise, there is not much they can do about it.

Reply:
I've added a new commit to the PR that adds a fallback to `getName()`.